packages/quantic/README.md

```diff
@@ -50,6 +50,26 @@ Start the server.
 - You can also run this command from the command line. `sfdx force:lightning:lwc:start`
 - View the server at http://localhost:3333/.
 
+## Run Cypress for Quantic components
```

Comment:
@acarpentier01 As you can see the documentation regarding Cypress tests is very basic. I went with something similar to [atomic](https://github.com/coveo/ui-kit/tree/master/packages/atomic#run-cypress-for-atomic-components). Please let me know if you feel more details should be added.